<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>植物大战僵尸外挂 by k4jar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="max_sun">
         <property name="text">
          <string>无限阳光</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kill_js">
         <property name="text">
          <string>秒杀僵尸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ban_cd">
         <property name="text">
          <string>禁用CD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="god_zw">
         <property name="text">
          <string>植物无敌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="runing">
         <property name="text">
          <string>后台运行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="god_js">
         <property name="text">
          <string>僵尸无敌</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="max_money">
         <property name="text">
          <string>无限金钱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="off_all">
         <property name="text">
          <string>一键取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chat">
         <property name="text">
          <string>联系作者</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff0004;&quot;&gt;仅用于学习娱乐,切勿用于非法用途!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff0004;&quot;&gt;请于下载后24小时内删除,使用者承担所有法律责任!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
